backend/api/main.py

```diff
@@ -530,112 +575,87 @@ async def get_match_status(
         raise HTTPException(status_code=404, detail="Datapoint not found!")
 
     async with aiohttp.ClientSession() as session:
-        response = await session.get(
-            f"{ORION_URL}/v2/entities/{row['entity_id']}/attrs/{row['attribute_name']}/?type={row['entity_type']}"
+        entity_id = row['entity_id']
+        attribute_name = row['attribute_name']
+        entity_type = row['entity_type']
+        url = f"{ORION_URL}/v2/entities/{entity_id}/attrs/{attribute_name}/?type={entity_type}"
+        response = await session.get(url)
+        match_status = response.status == 200
+        logging.info(f"Checking match status for entity_id: {entity_id}, attribute_name: {attribute_name}, entity_type: {entity_type}")
+        logging.info(f"Request URL: {url}")
+        logging.info(f"Response status: {response.status}")
+        return match_status
+
+async def check_and_update_connected(object_id: str, conn: asyncpg.Connection):
```

Comment:
If I see it correctly, here it will not check whether the entity and attribute actually exist in FIWARE right? For that, you can implement a test and send a request to Orion. Check GET /v2/entities/{entityId}/attrs/{attrName} in below. Please remember to also use `entity_type` as request parameter
https://swagger.lab.fiware.org/?url=https://raw.githubusercontent.com/FIWARE/specifications/master/OpenAPI/ngsiv2/ngsiv2-openapi.json#/Attributes/Get%20attribute%20data

Reply:
I have updated it in check_and_update_connected function